engine/flink/management/src/main/scala/pl/touk/nussknacker/engine/management/FlinkRestManager.scala

```diff
@@ -107,6 +112,29 @@ class FlinkRestManager(config: FlinkConfig, modelData: ModelData, mainClassName:
     }
   }
 
+  override def cancel(processName: ProcessName, user: User): Future[Unit] = {
+    def doCancel(overview: JobOverview) = {
+      val status = mapJobStatus(overview)
+      if (processStateDefinitionManager.statusActions(status).contains(ProcessActionType.Cancel) && isNotFinished(overview)) {
+        cancel(ExternalDeploymentId(overview.jid))
+      } else {
+        logger.warn(s"Trying to cancel ${processName.value} which is in status $status")
```

Comment:
`s"Trying to cancel terminated process ${processName.value} which is in status $status".` ?